vb-fe/src/components/player/PlayerWrapper.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FunctionComponent, useEffect } from "react";
+import { useDispatch } from "react-redux";
+import { usePlayerDevice, useErrorState, usePlaybackState } from "core/player/index";
```

Comment:
imports

Reply:
hella sus this isnt the latest version of the code